src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -69,6 +72,7 @@ public FilledRegionTypes(IEnumerable<PortModel> inPorts, IEnumerable<PortModel>
     [NodeCategory("Revit.Filter.RuleType")]
     [NodeDescription("FilterTypeSelectorDescription", typeof(DSRevitNodesUI.Properties.Resources))]
     [IsDesignScriptCompatible]
+    [OutPortTypes("string")]
```

Comment:
Hi @QilongTang , here are some nodes have "string" as output, I just add this, but don't work. Any suggestions?

Reply:
Right because right now, there are too many nodes with type as string. We will most likely enable the suggestion later with machine learning